en/docs/troubleshoot/faq.md

```diff
@@ -0,0 +1,147 @@
+# Frequently Asked Questions
+
+## General
+
+### Q: What is Choreo?
```

Comment:
Do we need to have the 'Q' in every question? Isn't the heading style that is applied sufficient to identify questions? WDYT?

Reply:
No Generally since it's a question, it  is grammatically incorrect if the ? ,mark isn't there.. also, Q: is used here more as an identifier... so  I think the question mark is indeed needed.

Reply:
What I meant was the use of Q. . .The question mark is definitely required.

Reply:
That makes it different from a title doesn't it?